### ISSUE TITLE
Unpredictable device visibility

### ISSUE THREAD

**Issue by bboettcher3** [libmapper/webmapper#36]
I've seen that for both on Ubuntu and Windows, some devices (T-Sticks) will show up and disappear unpredictably. To evaluate this a bit more, I used the python bindings to see what was present on the graph. As you can see below, I poll three times in a row for 1 second each, then print the devices on the network. 
![image](https://user-images.githubusercontent.com/13266420/204644416-0574343f-37cc-4157-8bd9-d3c797a40eed.png)

T-Stick 311 is seen the 1st time and 3rd time, but not the 2nd time. Aside from this trial, the device flashes in and out of webmapper in different states, sometimes with a few signals, sometimes with all 18, and sometimes with 0. I'm not sure what the cause of this might be. The T-Stick is sending OSC continuously throughout the trials, so the device is connected to the network, yet it's only seen intermittently by the python bindings.

@malloch any ideas what might be causing this?

**Comment by bboettcher3** [comment]
a webmapper screenshot when device appear with 0 signals:
![image](https://user-images.githubusercontent.com/13266420/204646479-bc017fe3-3f0a-43f1-ac7f-7ae1376c7010.png)

**Comment by bboettcher3** [comment]
Relating to this also, even when the device is visible in python (with 18 signals every time), webmapper may still show the device with 0 signals or no device at all, which probably implies a webmapper bug.

**Comment by malloch** [comment]
@bboettcher3 can you check whether you see the same behaviour from testmonitor.cpp (in the libmapper test suite)?  Perhaps the T-Stick is being inconsistent sending heartbeat pings. Are any other devices disappearing?

**Comment by bboettcher3** [comment]
@malloch I tried with test.exe and my own max test devices, they never disappear. I'd bet you're right about the heartbeat, but how would you explain the varying 0/11/18 signals? Is that a part of the heartbeat?

**Comment by DocSunset** [comment]
I have also observed behavior fitting this description.

Be sure to monitor your T-Stick's USB serial output (e.g. `screen /dev/ttyUSBx`). If it's a problem in the T-Stick firmware end of things there may be clues there.

**Comment by johnty** [comment]
My guess for this behaviour has always been dropped packets over WiFi. I've seen this happen between laptops over WiFi, but not over ethernet... I also recalled differences in reliability depending on networking environment, and the router being used. And on the ESP32 it was always more unreliable as well - there were some routers where the ESP32 device would simply not show up at all while on others it did.

**Comment by bboettcher3** [comment]
> I have also observed behavior fitting this description.
> 
> Be sure to monitor your T-Stick's USB serial output (e.g. `screen /dev/ttyUSBx`). If it's a problem in the T-Stick firmware end of things there may be clues there.

Yup, did this with no output clues, all seems to be running smoothly.
